test/tests/ver-1a/tests

```diff
@@ -1,11 +1,18 @@
 [Tests]
   design = 'EnclosureSinkScalarKernel.md PressureReleaseFluxIntegral.md EquilibriumBC.md'
```

Comment:
```suggestion
  design = 'EnclosureSinkScalarKernel.md PressureReleaseFluxIntegral.md EquilibriumBC.md'
  verification = 'ver-1a.md`
```

This syntax is used to designate the test as one for verification, and then set up linking to the documentation from the SQA V&V report.